src/main/kotlin/org/elaastic/questions/player/components/sequenceInfo/SequenceInfoResolver.kt

```diff
@@ -23,95 +23,116 @@ import org.elaastic.questions.assignment.sequence.Sequence
 import org.elaastic.questions.assignment.sequence.State
 import org.elaastic.questions.assignment.sequence.interaction.InteractionType
 import org.elaastic.questions.controller.MessageBuilder
-import java.lang.IllegalStateException
+import org.springframework.stereotype.Component
```

Comment:
not used

Reply:
Fix in 3b399d4871ea7e8964b875d608b780eb604afb92